src/Refactoring-Core/RBCondition.class.st

```diff
@@ -247,11 +258,12 @@ RBCondition class >> isClass: aRBClass definedIn: aRBNamespace [
 { #category : 'instance creation' }
 RBCondition class >> isClassNamed: className definedIn: aModel [
 
-	^ self new 
-		block: [ | aClassOrTrait |
-				aClassOrTrait := aModel classNamed: className.
-				aClassOrTrait isNotNil ]
-		errorString: [ className , ' is <1?:not > defined' ]
+	^ self new
+		  block: [
+			  | aClassOrTrait |
+			  aClassOrTrait := aModel classNamed: className.
+			  aClassOrTrait isNotNil ]
+		  errorString: className , ' is <1?:not >defined'
 ]
```

Comment:
Why you couldn't just use this `isClassNamed:definedIn:`?
Why did you have to duplicate and create `isClass:alreadyExistIn:`?